src/parser/shaman/enhancement/modules/talents/LashingFlames.tsx

```diff
@@ -51,6 +70,24 @@ class LashingFlames extends Analyzer {
     this.buffedFlameShockDmg += calculateEffectiveDamage(event, LASHING_FLAMES_BONUS);
   }
 
+  suggestions(when: When) {
+    when(this.uptimeThreshold).addSuggestion((suggest, actual, recommended) => suggest(
+      (
+        <>
+          Your <SpellLink id={SPELLS.LASHING_FLAMES_TALENT.id} /> uptime can be improved.
+          As <SpellLink id={SPELLS.LASHING_FLAMES_DEBUFF.id} /> increases the damage of <SpellLink id={SPELLS.FLAME_SHOCK.id} /> on its target,{` `}
+          try to maintain 100% uptime for maximum damage increase.
+          To achieve this, you can strike the target with <SpellLink id={SPELLS.LAVA_LASH.id} />{` `}
+          when <SpellLink id={SPELLS.LASHING_FLAMES_DEBUFF.id} /> is about to expire.
+        </>
+      )
+    )
+      .icon(SPELLS.LASHING_FLAMES_TALENT.icon)
+      .actual(<><SpellLink id={SPELLS.LASHING_FLAMES_DEBUFF.id} /> was active for {(actual*100).toFixed(2)}% of the fight</>)
+      .recommended('recommended: 99%')
+    )
```

Comment:
You can use ``formatPercentage(actual,)`` for the actual as it defaults to 2 decimal precision, and you can use ``recommended`` for the recommended value like ``.recommended(>{recommended}% is recommended)``

Reply:
@Pewtro you're right :+1:. I implemented your recommendation.